core/embed/rust/src/ui/model_mercury/layout.rs

```diff
@@ -2195,6 +2195,10 @@ pub static mp_module_trezorui2: Module = obj_module! {
     /// def show_wait_text(message: str, /) -> LayoutObj[None]:
     ///     """Show single-line text in the middle of the screen."""
     Qstr::MP_QSTR_show_wait_text => obj_fn_1!(new_show_wait_text).as_obj(),
+
+    /// def flow_get_address() -> LayoutObj[UiResult]:
+    ///     """Get address / receive funds."
```

Comment:
Missing quotes messing with IDE features
`"""Get address / receive funds."""`

Reply:
c741b2baba430ed3caad803c52c333fae9b7e9c4 ... wonder why style check didn't catch it